stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -223,6 +282,8 @@ public struct AnySequence<Element> : SequenceType {
   @available(*, unavailable, renamed="Element")
   public typealias T = Element
 
+  public typealias SubSequence = AnySequence<AnySequence.Generator.Element>
```

Comment:
I had that initially but it was ambiguous with the `SequenceType` protocol extension implementation.